library/tasks/TagSortBam.wdl

```diff
@@ -24,7 +24,7 @@ task CellSortBam {
   command {
     set -e
 
-    samtools sort -t GE -t UB -t CB -o cell-sorted.bam "${bam_input}"
+    TagSortBam -i ${bam_input} -o cell-sorted.bam -t GE -t UB -t CB
```

Comment:
I'm slightly confused by the order of tags here, since the description for the `CellSortBam` task is `"Sort bam_input by cell, then molecule, then gene."`

Reply:
Good catch - I've changed the order to match the description. Depending on the tool being used, there are cases where you would need to list things in the reverse order to the actual sort order you want. But that is definitely not the case for the sctools sorting tool.

Reply:
That's good to know! Thanks!

Reply:
It looks like there are a few tests that may need to be updated as well!